release_notes/ocp_3_7_release_notes.adoc

```diff
@@ -1839,11 +1839,22 @@ created with lowercase letters.
 later) the default values for `clusterNetworkCIDR` and `hostSubnetLength`
 changed. If the inventory file did not specify corresponding inventory
 variables, the upgrade would fail. This caused the controller service to not
-start back up. This bug fix requires that the inventory variables be set before
-upgrading or installing. As a result, if the required inventory variables are
-not set, the upgrade or installation will stop and tell the administrator to set
-the variables.
-(link:https://bugzilla.redhat.com/show_bug.cgi?id=1451023[*BZ#1451023*])
+start back up. The following Ansible variables must now be set in your inventory
+file before upgrading or installing:
++
+--
+- `osm_cluster_network_cidr`
+- `osm_host_subnet_length`
+- `openshift_portal_net`
+--
++
+These variables should match what is currently used in the cluster. If you do
+not remember what these values are, you can find them in
+*_/etc/origin/master/master-config.yaml_* on a master with the names
+`clusterNetworkCIDR` (for `osm_cluster_network_cidr`), `hostSubnetLength` (for
+`osm_host_subnet_length`), and `serviceNetworkCIDR` (for
+`openshift_portal_net`). These variables are automatically set for masters when
+using the scale-up playbook. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1451023[*BZ#1451023*])
```

Comment:
BZ#1451023 is already fixed, the above Ansible variables setting requirement is not needed any more. So this part should be removed now.

Reply:
@jianlinliu Follow up via https://github.com/openshift/openshift-docs/pull/8791.